lib-utilities/services/auth.go

```diff
@@ -42,10 +43,15 @@ func IsAuthorized(sessionToken string, privileges, oemPrivileges []string) (int3
 		},
 	)
 	if err != nil && response == nil {
-		log.Printf("error: something went wrong with rpc call: %v", err)
-		return http.StatusInternalServerError, errResponse.InternalError
+		errMsg := fmt.Sprintf("error: something went wrong with rpc call: %v", err)
```

Comment:
use `rpc call failed` instead of `something went wrong with rpc call`

Reply:
Done